replayer/frame.cpp

```diff
@@ -187,9 +187,36 @@ void readTail(
   }
 }
 
+// The boolean array better be divisible by 8
+std::vector<uint8_t> bool_to_bytes(std::vector<bool> arr) {
+  std::vector<uint8_t> ret;
+  ret.resize(arr.size() / 8);
+  for (size_t i=0; i<arr.size(); i++) {
+    ret[i / 8] |= arr[i] << (i % 8);
+  }
+  return ret;
+}
+std::vector<bool> bytes_to_bool(std::vector<uint8_t> arr) {
+  std::vector<bool> ret;
+  ret.resize(arr.size() * 8);
+  for (size_t i=0; i<arr.size(); i++) {
+    for (size_t k=0; k<8; k++) {
+      ret[i * 8 + k] = (arr[i] >> k) & 1;
+    }
+  }
+  return ret;
+}
+
```

Comment:
My only reasoning is that it's easier to do `creep_map[h * width + w]` instead of adding a whole new function to do this. But I could instead just add a new function. What do you thinking about adding the map width/height info to each frame @jgehring @syhw ?